.travis.yml

```diff
@@ -112,6 +113,8 @@ script:
   - python -c 'import numpy as np; print("relaxed strides checking:", np.ones((10,1),order="C").flags.f_contiguous)'
   # Make sure that relaxed strides checking is actually in effect; otherwise fail loudly
   - if [ "$NPY_RELAXED_STRIDES_CHECKING" == "1" ]; then python -c'import numpy as np; assert np.ones((10,1),order="C").flags.f_contiguous'; fi
+  # Test that mpmath actually uses gmpy2
+  - python -c 'import mpmath.libmp; assert mpmath.libmp.BACKEND == "gmpy"'
```

Comment:
Not sure if this is too strict: test suite may well pass without gmpy2. (Bikeshedding, non blocking comment).